stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js

```diff
@@ -0,0 +1,19 @@
+describe('getPerfectNumbers', function() {
+    it('should return a 0', function() {
+
+        var n = 6;
+        
+        var res = getPerfectNumbers(n);
+
+        expect(res).toBe(n + ' is a perfect number');
+    });
+
+    it('should return an error if is not a perfect number', function() {
```

Comment:
WARN! is not true that the function should return an error. instead it should return a message "<n> is not a perfect number". it must be corrected!

TODO: change the it('should... by the following:

it('should return "is not a perfect number" for a non-perfect number input') {